components/basic-analyzers/src/test/java/org/datacleaner/job/builder/AnalyzerComponentBuilderTest.java

```diff
@@ -57,15 +57,15 @@ public void testNoOriginatingTableBecauseOfMockColumns() throws Exception {
     }
 
     public void testBuildMultipleJobsForEachTable() throws Exception {
-        AnalyzerComponentBuilder<StringAnalyzer> jobBuilder = ajb.addAnalyzer(StringAnalyzer.class);
+        final AnalyzerComponentBuilder<StringAnalyzer> jobBuilder = ajb.addAnalyzer(StringAnalyzer.class);
 
-        Table table1 = new MutableTable("table1");
+        final Table table1 = new MutableTable("table1");
         jobBuilder.addInputColumn(new MetaModelInputColumn(
                 new MutableColumn("foo", ColumnType.VARCHAR, table1, 0, true)));
         jobBuilder.addInputColumn(new MetaModelInputColumn(
                 new MutableColumn("bar", ColumnType.VARCHAR, table1, 1, true)));
 
-        Table table2 = new MutableTable("table2");
+        final Table table2 = new MutableTable("table2");
```

Comment:
Very minor, and don't take them out now, but IMHO final fixes in irrelevant code do more harm than good: They take focus away from the real changes.